docs/src/model_setup/output_writers.md

```diff
@@ -158,6 +158,36 @@ NetCDFOutputWriter scheduled on IterationInterval(1):
 └── file size: 17.8 KiB
 ```
 
+`NetCDFOutputWriter` can also be configured for `outputs` that are interpolated or regridded to a different grid than `model.grid`.
+To use this functionality, the `output_grid` must be passed explicitly when constructing `NetCDFOutputWriter` along with the regridded / interpolated `outputs`.
+
+```jldoctest
+using Oceananigans
+using Oceananigans.Fields: interpolate!
+
+grid = RectilinearGrid(size=(1, 1, 8), extent=(1, 1, 1));
+model = NonhydrostaticModel(; grid)
+
+coarse_grid = RectilinearGrid(size=(grid.Nx, grid.Ny, grid.Nz÷2), extent=(grid.Lx, grid.Ly, grid.Lz))
+coarse_u = Field{Face, Center, Center}(coarse_grid)
+
+interpolate_u(model) = interpolate!(coarse_u, model.velocities.u)
+outputs = (; u = interpolate_u)
+
+output_writer = NetCDFOutputWriter(model, outputs, coarse_grid;
+                                   filename = "coarse_u.nc",
+                                   schedule = IterationInterval(1))
```

Comment:
Here `u = interpolate_u` is a function. Isn't this failing without passing dimensions for the output? That's why my original example output a `Field` directly and updated it with a `Callback`.

Reply:
Ah, nvmd. I just saw that @glwagner added new functionality that infers dimensions from functions that output `Field`s. Nice!